templates/primary-secondary/primary-secondary.js

```diff
@@ -885,6 +893,10 @@ class TemplatePrimarySecondary extends RtlMixin(LitElement) {
 		};
 	}
 
+	_computeSizeKey() {
+		return `d2l-primary-secondary-${window.location.href}`;
```

Comment:
I'm not sure how well this will work in practice... most URLs contain the orgUnitId in the query-string or even worse as part of the URL itself. Wonder if we want to introduce something like a `memory-key` attribute or something that could be used for this kind of thing, i.e. `<d2l-template-primary-secondary memory-key="quick-eval">`?

If this came up a few times, we could even throw it in a mixin.

Reply:
Good call, having to add a property for it is unfortunate but probably for the best.